docs/HoloLens/Occlusion.md

```diff
@@ -0,0 +1,25 @@
+---
+id: Occlusion
+title: Occlusion
+---
+
+The HoloLens is capable of using certain parts of the environment as occlusion to hide Holograms. 
+Depending on your experience and its use case, there's pros and cons to the effects occlusion provides.
+
+Each occlusion option can be configured on a per-experience basis, through the experience's Scene settings in Enklu Cloud.
+
+### Space Occlusion
+
+When an experience is using a <a href='/Guide_Spaces.html'>Space</a>, the mesh visible in the Cloud Editor can be used for occlusion.
+This is useful in scenarios where multiple rooms are filled with content and a user isn't meant to see through walls, thus increasing immersion.
+
+This feature starts <b>enabled</b> by default, and may be beneficial to disable in experience that are meant to display large content that is physically too big to fit in a room normally.
```

Comment:
"and it may be beneficial to disable Space Occlusion in an experience with content that is physically too big to fit in a room normally."